firebase-config-interop/src/main/java/com/google/firebase/remoteconfig/interop/rollouts/RolloutAssignment.java

```diff
@@ -57,10 +57,8 @@ public abstract class RolloutAssignment {
   public static final DataEncoder ROLLOUT_ASSIGNMENT_JSON_ENCODER =
       new JsonDataEncoderBuilder().configureWith(AutoRolloutAssignmentEncoder.CONFIG).build();
 
-  /** Create a {@code RolloutAssignment} from its JSON-encoded representation. */
   @NonNull
-  public static RolloutAssignment create(@NonNull String jsonString) throws JSONException {
-    final JSONObject json = new JSONObject(jsonString);
+  public static RolloutAssignment create(@NonNull JSONObject json) throws JSONException {
```

Comment:
Nit: maybe add a description for this `create` method too?

Reply:
good call